src/test/resources/TestNG.xml

```diff
@@ -6,9 +6,9 @@
 
     <test name="Example Test" preserve-order="false">
         <classes>
-            <class name="Homework17" />
-<!--            <class name="ProfileTests"/>-->
-<!--            <class name="BaseTest"/>-->
+            <class name="UpdateHW16" />
+            <class name="ProfileTests"/>-->
+            <class name="BaseTest"/>-->
         </classes>
     </test>
 </suite>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/211878378-e36502b4-d3a6-4b05-8d0f-047b7a7cb9d4.png)

For example

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
<suite name="TestNG">
    <parameter name="BaseURL" value="https://bbb.testpro.io/" />
    <test name="Example Test" preserve-order="false">
        <classes>
            <class name="UpdateHW16" />
        </classes>
    </test>
</suite>
```